docs/source/examples/predict_next_purchase.ipynb

```diff
@@ -0,0 +1,400 @@
+    "    cutoff_time=lt,\n",
+    "    cutoff_time_in_index=True,\n",
+    "    include_cutoff_time=False,\n",
+    "    verbose=False,\n",
+    ")\n",
+    "\n",
+    "fm.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "There are two outputs from DFS: a feature matrix and feature definitions. The feature matrix is a table that contains the feature values based on the cutoff times from our labels. Feature definitions are features in a list that can be stored and reused later to calculate the same set of features on future data.\n",
+    "\n",
+    "## Machine Learning\n",
+    "\n",
+    "In the previous steps, we generated the labels and features. The final step is to build the machine learning pipeline.\n",
+    "\n",
+    "### Splitting the Data\n",
+    "\n",
+    "We will start by extracting the labels from the feature matrix and splitting the data into a training set and holdout set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y = fm.pop('bought_product')\n",
+    "splits = evalml.preprocessing.split_data(fm, y, test_size=0.2, random_state=0)\n",
+    "X_train, X_holdout, y_train, y_holdout = splits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Finding the Best Model\n",
+    "\n",
+    "Then, we run a search on the training set for the best machine learning model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl = evalml.AutoMLSearch(problem_type='binary', objective='f1', random_state=0)\n",
+    "automl.search(X_train, y_train, data_checks='disabled', show_iteration_plot=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the search is complete, we can print out information about the best pipeline found, such as the parameters in each component. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl.best_pipeline.describe()\n",
+    "automl.best_pipeline.graph()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's score the model performance by evaluating predictions on the holdout set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_pipeline = automl.best_pipeline.fit(X_train, y_train)\n",
+    "score = best_pipeline.score(X_holdout, y_holdout, objectives=['f1'])\n",
+    "dict(score)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Next Steps\n",
+    "\n",
+    "At this point, we have completed the machine learning application. We can revisit each step to explore and fine-tune with different parameters until the model is ready for deployment."
```

Comment:
@jeff-hernandez these two examples you've added are great!! Its really exciting to have evalml included here 😁 👏 

I had one suggestion: you could add something at the end like this
> For more information on how to work with the models produced by EvalML, take a look at [the EvalML documentation](https://evalml.alteryx.com/en/stable/index.html).

I know you already linked to the evalml docs at the top, but it could be nice to call this out for people who skim to the bottom and are left wondering what else they can do.